File: Server/src/main/java/com/babob/sporcantam/Seller/Seller.java
package com.babob.sporcantam.Seller;
import com.babob.sporcantam.User.User;
import javax.persistence.Entity;
;

@Entity
public class Seller extends User{
    private String company_name;
    private String phone_number;
    private String company_address;
    private String IBAN;

    public String getCompany_address() {
        return company_address;
    }

    public String getIBAN() {
        return IBAN;
    }

    public String getPhone_number() {
        return phone_number;
    }

    public String getCompany_name() {
        return company_name;
    }

    public void setCompany_address(String company_address) {
        this.company_address = company_address;
    }

    public void setCompany_name(String company_name) {
        this.company_name = company_name;
    }

    public void setIBAN(String IBAN) {
        this.IBAN = IBAN;
    }

    public void setPhone_number(String phone_number) {
        this.phone_number = phone_number;
    }

    @Override
    public String toString() {
        return super.toString() + "\nCompany Name: "+ company_name + "\nCompany Address: " + company_address +
                "\nIBAN: "+ IBAN + "\nPhone Number: "+ phone_number +"\n";
    }
}

File: Mobile/Android/app/src/main/java/com/babob/sporcantam/item/Seller.kt
package com.babob.sporcantam.item

import java.io.Serializable

class Seller:Serializable {
    var firstName:String = ""
    var lastName:String = ""
    var address:String = ""
    var IBAN:String = ""
    var phoneNumber:String = ""
    var email:String=""

    constructor(firstName:String, lastName:String, address:String, IBAN:String, phoneNumber:String, email:String){
        this.firstName=firstName
        this.lastName=lastName
        this.address=address
        this.IBAN=IBAN
        this.phoneNumber=phoneNumber
        this.email=email
    }

    constructor()
}